README.md

```diff
@@ -75,6 +64,65 @@ The unit tests have been implemented by sending a POST request to GitHub and obs
 One test case send valid API data and expects a valid response, another sends invalid API data and
 expects an invalid response.
 
+## API Documentation
+
+### POST requests
+`/push-events`
+
+#### Description
+Pushes an event to the server
+
+#### Parameters
+None
+
+Request body implemented according to [this](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push)
```

Comment:
Yes it is implemented according to that, but the properties our CI server supports are the one in your example below.